### PR TITLE
fix(halyard/custom): s/Lifecycle/LifeCycle/

### PR DESCRIPTION
**Pull request by lwander**
https://github.com/spinnaker/spinnaker/issues/2851

### CODE REVIEW

File: reference/halyard/custom.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ cluster.
 | `port` | The port number this service is bound to, and will accept requests on. |
 | `safeToUpdate` | Whether or not this service can be shutdown, and spun on a new VM/container. This protects datastores like Vault & Redis from being taken down from Halyard. |
 | `scheme` | The URI scheme used to address this service, e.g. `http` vs.  `https`. |
-| `skipLifecycleManagement` | Whether or not Halyard should skip managing a service's lifecycle. |
+| `skipLifeCycleManagement` | Whether or not Halyard should skip managing a service's life cycle. |
 | `targetSize` | The initial number of nodes this service will be created with.  This is only respected on the initial deployment, and further edits will be rejected in favor of the prior service version's size. |
```
Comment (dorbin) [inline reference/halyard/custom.md:91]: Actually, [one word](https://developers.google.com/style/word-list#letter-L).

Comment (lwander) [reply reference/halyard/custom.md:91]: Darn....

If we put it back into one word, is it jarring that the flag has the C capitalized?

Comment (dorbin) [reply reference/halyard/custom.md:91]: Not to me. 
IMO, we should do whatever has the least negative impact on the user. If [things work better](https://github.com/spinnaker/spinnaker/issues/2851) when it's skipLifeCycleManagement, I'm ok with violating the style in doc text.

Comment (lwander) [reply reference/halyard/custom.md:91]: Ok, let's keep it as-is for now

**Pull request merged**